iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -52,11 +49,6 @@ class ConversationMessagesCell: UICollectionViewCell {
         self.collectionView.clipsToBounds = false
         self.contentView.addSubview(self.collectionView)
 
-        self.collectionView.contentInset = UIEdgeInsets(top: Theme.contentOffset,
-                                                        left: 0,
-                                                        bottom: 0,
-                                                        right: 0)
-
```

Comment:
This content inset isn't needed. Insets don't work with the timeline layout anyway.

Reply:
Nice